assets/js/test-end.js

```diff
@@ -128,10 +128,13 @@
                     assert, i,
                     reportHTML = '',
                     header,
+                    failingModules = '',
                     expression,
                     submitUrl = "https://github.com/moment/moment/issues/new",
                     searchUrl = "https://github.com/moment/moment/search",
                     titleText = "" + failures + " test" + (failures !== 1 ? "s" : "") + " failed. ",
+                    shortenedTitle = "" + failures + " test" + (failures !== 1 ? "s" : "") + " failed in modules: ",
```

Comment:
Changed everything to single quotes.

Reply:
On second thought, for the strings containing the html tags starting on line 198 we need double quotes, otherwise the single quotes for `class='button'` need to be escaped.

I've changed these to double quotes but now it's still inconsistent. Shall I change everything to double quotes?

Reply:
Took me some time to pick this up, but done.